### PR TITLE
torqued: cache bucket points every minute while onroad

### PR DESCRIPTION
**Pull request by sunnyhaibin**
Currently, the bucket points are cached only when `signal.SIGINT` is triggered (i.e., going offroad). This works as expected for most physical setups, but does not work properly for physical setups that do not have [comma power](https://comma.ai/shop/comma-power.html) installed (the device would lose power immediately when the car is turned off due to the lack of constant 12V), preventing the points to get cached properly.

This change continuously caches the bucket points to `LiveTorqueParameters` every 60 seconds. If the car is turned off without comma power connected or does not have constant 12V being sent to the device, most of the points would at least be cached to `LiveTorqueParameters` for future routes.

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
This is something we've been working on for a bit and want to support soon. I started a public GH project to track this, so external contributors can participate. https://github.com/orgs/commaai/projects/25

File: selfdrive/locationd/helpers.py [hunk 1]
```diff
@@ -70,8 +70,12 @@ def get_msg(self, valid: bool, with_points: bool) -> log.Event:
...
+  cache_points(param_name, estimator, True)
+  sys.exit(0)
+
+
+def cache_points(param_name, estimator, valid):
   cloudlog.warning(f"Caching {param_name} param")
   params = Params()
-  msg = estimator.get_msg(valid=True, with_points=True)
+  msg = estimator.get_msg(valid=valid, with_points=True)
   params.put(param_name, msg.to_bytes())
```
Comment (adeebshihadeh) [inline selfdrive/locationd/helpers.py:81]: needs to use put_nonblocking if we're writing while onroad, otherwise you're vulnerable to slow disk io

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]
Let's remove the caching on exit and just stick to writing while onroad

File: selfdrive/locationd/helpers.py (hunk 1 above)
Comment (sunnyhaibin) [reply selfdrive/locationd/helpers.py]: Updated.

File: selfdrive/locationd/helpers.py [hunk 2]
```diff
@@ -68,10 +68,7 @@ def get_msg(self, valid: bool, with_points: bool) -> log.Event:
     raise NotImplementedError
 
 
-def cache_points_onexit(param_name, estimator, sig, frame):
-  signal.signal(sig, signal.SIG_DFL)
+def cache_points(param_name, estimator, valid):
   cloudlog.warning(f"Caching {param_name} param")
```
Comment (adeebshihadeh) [inline selfdrive/locationd/helpers.py:72]: remove the log, it'll be too high volume now

**Pull request merged**